cpp/ql/test/library-tests/ir/ir/aliased_ssa_ir.qlref

```diff
@@ -0,0 +1 @@
+semmle/code/cpp/ir/implementation/aliased_ssa/PrintIR.ql
```

Comment:
Why re-introduce the `*aliased_ssa_ir.{qlref,expected}` files? I thought you had a good reason to delete those in #919.

Reply:
Oops, I re-added those by mistake. They're useful for seeing diffs caused by changes, but they're still not worth keeping up to date with every PR. Fixed.